### PR TITLE
data release

### PR DESCRIPTION
**Pull request by n-h-diaz**
auto1d_2023_07_11_21_10_22
frequent_2023_07_11_20_02_25
auto1w_2023_07_09_20_57_22
auto2w_2023_07_11_10_41_01
ipcc_2023_07_01_08_22_56
biomedical_2023_07_02_03_45_18
disaster_2023_07_09_21_31_47
country_2023_07_11_22_29_43
infrequent_2023_07_10_11_27_44
place_2023_07_03_03_47_46

datcom-store.dc_kg_2023_07_11_06_02_09

### CODE REVIEW

File: test/statvar_ranking/missing_Earth_country_rankings.json [hunk 1]
```diff
@@ -554,18 +554,6 @@
...
-    "statsVars": [
-      "Count_Person_Employed"
-    ],
-    "relatedChart": {
-      "scale": true,
-      "statsVars": [
-        "Count_Person_Employed"
-      ]
-    }
-  },
```
Comment (juliawu) [inline test/statvar_ranking/missing_Earth_country_rankings.json:568]: Is removing this test case expected?

File: internal/server/v1/variables/golden/bulk_variables/california_and_santa_clara_union.json [hunk 2]
```diff
@@ -25683,58 +25683,6 @@
...
-        "MaxTemp_Daily_5PctProb_Greater_Atleast1DayAYear_CMIP6_GFDL-CM4_SSP585",
-        "MaxTemp_Daily_5PctProb_Greater_Atleast1DayAYear_CMIP6_GFDL-ESM4_SSP245",
-        "MaxTemp_Daily_5PctProb_Greater_Atleast1DayAYear_CMIP6_GFDL-ESM4_SSP585",
-        "MaxTemp_Daily_5PctProb_Greater_Atleast1DayAYear_CMIP6_HADGEM3-GC31-LL_SSP245",
-        "MaxTemp_Daily_5PctProb_Greater_Atleast1DayAYear_CMIP6_HADGEM3-GC31-LL_SSP585",
-        "MaxTemp_Daily_5PctProb_Greater_Atleast1DayAYear_CMIP6_HADGEM3-GC31-MM_SSP585",
-        "MaxTemp_Daily_5PctProb_Greater_Atleast1DayAYear_CMIP6_MPI-ESM1-2-HR_SSP245",
-        "MaxTemp_Daily_5PctProb_Greater_Atleast1DayAYear_CMIP6_MPI-ESM1-2-HR_SSP585",
-        "MaxTemp_Daily_5PctProb_Greater_Atleast1DayAYear_CMIP6_MPI-ESM1-2-LR_SSP245",
-        "MaxTemp_Daily_5PctProb_Greater_Atleast1DayAYear_CMIP6_MPI-ESM1-2-LR_SSP585",
```
Comment (juliawu) [inline internal/server/v1/variables/golden/bulk_variables/california_and_santa_clara_union.json:25737]: Are these deletions expected?

File: internal/server/v1/variables/golden/bulk_variables/california_and_santa_clara_union.json [hunk 3]
```diff
@@ -28109,7 +28057,6 @@
         "Mean_UsualHoursWorked_Person_WorkedInThePast12Months",
         "Mean_VaporPressureDeficit",
         "Mean_VaporPressureDeficit_Forest",
-        "Mean_Visibility",
```
Comment (juliawu) [inline internal/server/v1/variables/golden/bulk_variables/california_and_santa_clara_union.json:28112]: Why is this being removed?

File: internal/server/v0/placestatvar/golden/get_place_stat_vars/santa_clara.json [hunk 4]
```diff
@@ -15350,58 +15350,6 @@
...
-        "MaxTemp_Daily_5PctProb_Greater_Atleast1DayAYear_CMIP6_GFDL-CM4_SSP585",
-        "MaxTemp_Daily_5PctProb_Greater_Atleast1DayAYear_CMIP6_GFDL-ESM4_SSP245",
-        "MaxTemp_Daily_5PctProb_Greater_Atleast1DayAYear_CMIP6_GFDL-ESM4_SSP585",
-        "MaxTemp_Daily_5PctProb_Greater_Atleast1DayAYear_CMIP6_HADGEM3-GC31-LL_SSP245",
-        "MaxTemp_Daily_5PctProb_Greater_Atleast1DayAYear_CMIP6_HADGEM3-GC31-LL_SSP585",
-        "MaxTemp_Daily_5PctProb_Greater_Atleast1DayAYear_CMIP6_HADGEM3-GC31-MM_SSP585",
-        "MaxTemp_Daily_5PctProb_Greater_Atleast1DayAYear_CMIP6_MPI-ESM1-2-HR_SSP245",
-        "MaxTemp_Daily_5PctProb_Greater_Atleast1DayAYear_CMIP6_MPI-ESM1-2-HR_SSP585",
-        "MaxTemp_Daily_5PctProb_Greater_Atleast1DayAYear_CMIP6_MPI-ESM1-2-LR_SSP245",
-        "MaxTemp_Daily_5PctProb_Greater_Atleast1DayAYear_CMIP6_MPI-ESM1-2-LR_SSP585",
```
Comment (juliawu) [inline internal/server/v0/placestatvar/golden/get_place_stat_vars/santa_clara.json:15404]: Are these deletions expected?

File: internal/server/v0/placestatvar/golden/get_place_stat_vars/santa_clara.json [hunk 5]
```diff
@@ -17176,7 +17124,6 @@
         "Mean_UsualHoursWorked_Person_WorkedInThePast12Months",
         "Mean_VaporPressureDeficit",
         "Mean_VaporPressureDeficit_Forest",
-        "Mean_Visibility",
```
Comment (juliawu) [inline internal/server/v0/placestatvar/golden/get_place_stat_vars/santa_clara.json:17179]: Is this expected?

File: test/statvar_ranking/missing_Earth_country_rankings.json (hunk 1 above)
Comment (pradh) [reply test/statvar_ranking/missing_Earth_country_rankings.json:568]: I think this means we now have this SV... "missing_....json" is the file

File: internal/server/v1/variables/golden/bulk_variables/california_and_santa_clara_union.json (hunk 2 above)
Comment (pradh) [reply internal/server/v1/variables/golden/bulk_variables/california_and_santa_clara_union.json:25737]: This might be - @jehangiramjad

Comment (jehangiramjad) [reply internal/server/v1/variables/golden/bulk_variables/california_and_santa_clara_union.json:25737]: Yes, these should be expected. None of them in the path /resolved_mcf/raw_stat_vars/cmip6/vars.mcf

File: internal/server/statvar/golden/search_statvar/count_person.json [hunk 6]
```diff
@@ -3845,160 +3989,16 @@
       "dcid": "Count_Person_100OrMoreYears_WhiteAloneNotHispanicOrLatino"
     },
     {
-      "name": "Population: 10th Grade, Female",
```
Comment (ajaits) [inline internal/server/statvar/golden/search_statvar/count_person.json:3848]: I assume these deletions are due to sampling as we have a lot more Count_Person_* statvars now?

**Review by ajaits — APPROVED** [review]
new countries demographics added as expected.

**Pull request closed without merging**